### PR TITLE
Kapulimbs Parity Pt. 1

### PR DESCRIPTION
**Pull request by Tsar-Salat**
Port *from* (actual content may vary):
- https://github.com/tgstation/tgstation/pull/49062 (everything except the new mutations, actually.)
- https://github.com/tgstation/tgstation/pull/49771
- https://github.com/tgstation/tgstation/pull/52774
- https://github.com/tgstation/tgstation/pull/54525

## About The Pull Request

This is a complete species.dm refactor.

Included is refactors for organs, species features and surgery.

This removes the hacky bypass of #8505 and makes the code handle it automatically like any other organ/species feature.

I had to take extra care because our wing system is its own mutorgan, unlike TG. I tried my best, but if you see any issues on that front, lmk.

## Why It's Good For The Game

Will greatly improve our reach to kapulimbs and modsuits, as well as many other important systems.

Less code around the broken prototype, more towards the complete system.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Organ extraction, missing in health analyzer

Screenshot 2024-08-06 013853

Removing, then readding forced organs

Screenshot 2024-08-06 014431

</details>

## Changelog
:cl: rkz, Tralezab, Timberpoes, Ghommie
refactor: refactors species.dm organs and features
code: folded default_features into mutant_bodyparts
fix: fixed hacky override for lizards, now spines work in the preference menu from the get-go.
tweak: sensory regeneration no longer sprouts new tails
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
nows a good a time as any to post a reminder that i can be reached for help on bee's discord.

**Comment by Tsar-Salat** [comment]
I don't know what the fuck I'm doing

**Comment by Tsar-Salat** [comment]
I have an inkling of what I'm doing

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -231,164 +255,102 @@ GLOBAL_LIST_EMPTY(features_by_species)
...
+		ORGAN_SLOT_EYES,
+		ORGAN_SLOT_EARS,
+		ORGAN_SLOT_TONGUE,
+		ORGAN_SLOT_LIVER,
+		ORGAN_SLOT_STOMACH,
+		ORGAN_SLOT_WINGS
+	)
+
+	//if theres no added wing type, we want to avoid adding a null
+	if(isnull(mutantwings))
```
Comment (Tsar-Salat) [inline code/modules/mob/living/carbon/human/species.dm:298]: my glorious hacky wing if statement list removal shitcode that prevents everyone from having wings.

Thanks, 3 yr old zesko pr.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**